### PR TITLE
Jr deploy stuff

### PR DESCRIPTION
**Pull request by jacmrob**

### CODE REVIEW

File: configs/manifest.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+render "docker-compose.yaml.ctmpl"
+render "gpalloc.conf.ctmpl"
+copy_file "site.conf"
+
+copy_secret_from_path "secret/dsde/dsp-techops/common/server.crt"
+copy_secret_from_path "secret/dsde/dsp-techops/common/server.key"
+copy_secret_from_path "secret/dsde/dsp-techops/common/ca-bundle.crt"
+
+copy_secret_from_path "secret/dsde/firecloud/#{$env}/common/billing-account.pem", field = "private_key", output_file_name = "billing-account.pem"
```
Comment (helgridly) [inline configs/manifest.rb]: what populates `$env` here?

Comment (jacmrob) [reply configs/manifest.rb]: ah. setting ENV in the environment. which we will need to do in deploy.sh but does it matter which billing pem we use? can we use the one in dev? do we always want to use the one in dev?

Comment (helgridly) [reply configs/manifest.rb]: I think test.firecloud.org and quality.firecloud.org have different master billing accounts, no?

Comment (jacmrob) [reply configs/manifest.rb]: yeah. so i guess we do need to pass in an environment? where do you set that tho?

**Pull request merged**